### PR TITLE
feat: add analytics

### PR DESCRIPTION
**Pull request by efraa**

### CODE REVIEW

File: src/mods/projects/components/creation-editing/CreationEditingProject.tsx [hunk 1]
```diff
@@ -43,13 +44,15 @@ export const CreationEditingProject = () => {
             onSuccess() {
               onClose()
 
+              analytics.track('Project updated')
```
Comment (psanders) [inline src/mods/projects/components/creation-editing/CreationEditingProject.tsx:47]: Would it make sense to use a machine-readable string here instead? For example `project-updated`. I'm not suggesting we change this, but want to know the implications of changing the string.

**Review by psanders — APPROVED** [review]
LGTM.

File: src/mods/projects/components/creation-editing/CreationEditingProject.tsx (hunk 1 above)
Comment (efraa) [reply src/mods/projects/components/creation-editing/CreationEditingProject.tsx:47]: This is a tool for management, KPIs, marketing, or sales.

I could easily change it, I'm just saying this is just a human-friendly name that will show up on the board later.

Comment (efraa) [reply src/mods/projects/components/creation-editing/CreationEditingProject.tsx:47]: In the future, we may want to be more contextual with events, something like:

"Log in using Google, or Github provider"
"User tried to create more projects than allowed in his plan"
"User reaches his number limits"
"User created a secret of type JSON or Text"
"User made a test call from the UI"

Comment (efraa) [reply src/mods/projects/components/creation-editing/CreationEditingProject.tsx:47]: But if we want to change this into machine-readable, it's possible.

Comment (psanders) [reply src/mods/projects/components/creation-editing/CreationEditingProject.tsx:47]: No worries. Continue course with what makes sense for you.

**Pull request merged**